model/registry/http.yaml

```diff
@@ -95,6 +95,12 @@ groups:
           was the cause of the resending (e.g. redirection, authorization failure, 503 Server Unavailable, network issues,
           or any other).
         examples: 3
+      - id: request.size
+        type: int
+        brief: >
+          The total size of the request in bytes. This should be the total number of bytes sent over the wire, and includes all framing and control
```

Comment:
Feel free to ignore, but I think using `, including all...` gives better flow to the sentence

Reply:
made this change, though I think I left it still a bit awkward